src/checkUsersTry.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+function checkUsersTry(usersTry, numberToGuess) {
+  if (isNaN(+usersTry) || usersTry.length !== 4) {
+    return 'The number must contain only 4 digits!';
+  }
+
+  if (new Set(usersTry).size !== 4) {
+    return 'The number must contain only unique digits!';
+  }
+
+  if (usersTry === numberToGuess) {
+    return 'OK';
+  }
+
+  let bulls = 0;
+  let cows = 0;
+
+  for (let i = 0; i < usersTry.length; i++) {
```

Comment:
Try to avoid using old traditional `for` loops. Instead, use specific ones like filter/map/foreach/reduce. Here fits the best `reduce`.